<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgProject</class>
 <widget class="QDialog" name="DlgProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pg_description">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>718</width>
        <height>201</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Project Description</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QTextEdit" name="ed_description"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="ed_project_name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pg_connections">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>718</width>
        <height>201</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Connections</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Production Connection</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="bt_connections">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="pgdbatool.qrc">
           <normaloff>:/icons/images/icons/icons8-database-administrator-48.png</normaloff>:/icons/images/icons/icons8-database-administrator-48.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cb_production_connection"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Development Connection</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_development_connection"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cb_staging_connection"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Staging Connection</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>97</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Paths">
      <attribute name="label">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Project Path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ed_project_path"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Query Path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ed_query_dir"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Models Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ed_models_dir"/>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="bt_models_directory">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="pgdbatool.qrc">
           <normaloff>:/icons/images/icons/icons8-opened-folder-40.png</normaloff>:/icons/images/icons/icons8-opened-folder-40.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>129</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="bt_query_directory">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="pgdbatool.qrc">
           <normaloff>:/icons/images/icons/icons8-opened-folder-40.png</normaloff>:/icons/images/icons/icons8-opened-folder-40.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="bt_project_path">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="pgdbatool.qrc">
           <normaloff>:/icons/images/icons/icons8-opened-folder-40.png</normaloff>:/icons/images/icons/icons8-opened-folder-40.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ed_project_name</tabstop>
 </tabstops>
 <resources>
  <include location="pgdbatool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
